front/lib/actions/helpers.ts

```diff
@@ -65,7 +58,8 @@ export async function callAction<V extends t.Mixed>({
     logger
   );
 
-  const r = await prodAPI.runApp(user, app, config, [input]);
+  // The doc tracker apps are running without any group privileges for now.
```

Comment:
Why is this file referring to doc tracker ooc ?

Reply:
It's used only by doc tracker today. I assumed this was some old code that will never be used by something else.

Reply:
OK can we leave a `TODO(GROUPS_INFRA)` here as this will need to be addressed before release :pray: (exact syntax important so that we can search for them) :pray:

Reply:
Added here @spolu https://github.com/dust-tt/dust/pull/6940

Reply:
tx tx